src/spec/payid-easy-checkout.md

```diff
@@ -0,0 +1,378 @@
+* PayID Discovery Server: An endpoint that returns a PayID Discovery JRD conforming to the PayID Discovery Protocol as defined in [PAYID-DISCOVERY][].
+* Recipient: An individual or entity receiving a payment (e.g., e-commerce merchant, charity).
+* Payer: An individual or entity originating a payment to a `recipient`.
+* Wallet: A device or application that holds funds (may be a non-custodial wallet).
+* PayID Easy Checkout URL: A URL that is the result of the PayID Easy Checkout protocol; can be used to redirect a client to a wallet corresponding to a particular PayID as defined in [PAYID-URI][].
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC2119][] and [RFC9174][].
+
+# Introduction
+
+The PayID Easy Checkout Protocol is a minimal protocol that allows a recipient (e.g., an online merchant or a charity) to
+request a payment from a payer using only the payer's PayID as defined in [PAYID-URI][]. Implementations
+of this protocol require little to no server-side engineering efforts, while creating a seamless and uniform
+user experience for payers.
+
+The main focus of the protocol is on PayID Easy Checkout Discovery, which defines how a PayID Easy Checkout Client can 
+use a PayID to retrieve a PayID Easy Checkout URL which represents a resource that the payer's digital wallet can use to initiate 
+a payment to the merchant. 
+
+Though the [appendix](#appendix) of this specification provides an example usage of a 
+PayID Easy Checkout URL using Web Redirects, supplemental RFCs are needed to define the different ways in which a PayID
+client can utilize a PayID Easy Checkout URL.
+
+# PayID Easy Checkout Protocol
+The PayID Easy Checkout Protocol can be used to facilitate an end-to-end checkout flow between a payment recipient, such
+as an online merchant, and a payer via a sending client, such as a wallet.
+
+The protocol is comprised of two parts:
+
+1. PayID Easy Checkout Discovery
+2. PayID Easy Checkout URL Assembly
+
+## PayID Easy Checkout Discovery
+PayID Easy Checkout Discovery extends [PAYID-DISCOVERY][] by defining a new link in the PayID metadata JRD
+returned by a PayID Discovery query. This link, defined in the [JRD section](#payid-easy-checkout-jrds) 
+of this specification, includes the PayID Easy Checkout URL representing a resource on the wallet which can
+be used to complete a payment.
+
+Recipients who wish to initiate an Easy Checkout flow MUST query the sender's PayID Discovery Server to 
+obtain a PayID Easy Checkout URL. PayID Discovery Servers that wish to enable PayID Easy
+Checkout MUST include a JRD Link conforming to the definition in the [JRD section](#payid-easy-checkout-jrds) of this document
```

Comment:
```suggestion
Checkout MUST include an Easy Checkout JRD Link 
```

We already say, "Conforming to the JRD section of this document" above - I don't think we need to keep repeating that.